torch_xla/csrc/ops/symeig.cpp

```diff
@@ -20,7 +20,9 @@ std::vector<xla::XlaOp> LowerSymEig(const xla::XlaOp& input, bool eigenvectors,
   xla::XlaOp v = self_adj_eig_result.v;
   xla::XlaOp w = self_adj_eig_result.w;
   if (!eigenvectors) {
-    v = xla::Zeros(input.builder(), XlaHelpers::ShapeOfXlaOp(v));
+    v = xla::Zeros(input.builder(),
```

Comment:
Just use `xla::Zero()`

Reply:
Does it create an 0-numel `XlaOp` i.e., of shape `{0}`? I wasn't sure about it, so I went for the more verbose solution.

Reply:
Yep, that's what it does:

https://github.com/tensorflow/tensorflow/blob/be20d163c9be8991feeb6bb39abe31ab55346110/tensorflow/compiler/xla/client/lib/constants.h#L121